src/scripts/ci_build.py

```diff
@@ -684,8 +687,7 @@ def main(args=None):
             'src/editors/vscode/scripts/common.py',
             'src/editors/vscode/scripts/test.py']
 
-        full_paths = [os.path.join(root_dir, s) for s in py_scripts]
-        cmds.append([py_interp, '-m', 'pylint'] + pylint_flags + full_paths)
+        cmds.append(["indir:%s" % root_dir, py_interp, '-m', 'pylint'] + pylint_flags + py_scripts)
```

Comment:
The paths reported in the log must be relative to the repository root. To accomplish that without a custom script, we'll have to set CWD to the repo root when running `pylint`.

Reply:
Maybe add a comment here along these lines

Reply:
Haha! That's pure retaliation for https://github.com/randombit/botan/pull/4181#discussion_r1665689550. 😏

Reply:
😅